spec/std/http/server/handlers/static_file_handler_spec.cr

```diff
@@ -201,8 +201,8 @@ describe HTTP::StaticFileHandler do
         headers = HTTP::Headers{"Range" => "bytes=0-0"}
         response = handle HTTP::Request.new("GET", "/empty.txt", headers)
 
-        response.status_code.should eq(200)
-        response.headers["Content-Range"]?.should be_nil
+        response.status_code.should eq(416)
```

Comment:
This is no longer a valid Range header, so I'd suggest to move this spec outside this `describe`